Src/AmrCore/AMReX_AmrMesh.H

```diff
@@ -260,6 +264,11 @@ protected:
     Vector<DistributionMapping> dmap;
     Vector<BoxArray>            grids;
 
+#ifdef AMREX_USE_BITTREE
+    bool use_bittree;
```

Comment:
I think we should move this flag into `AmrInfo` and gives it a default value of false.  One constructor of AmrMesh does not use ParmParse at all and it relies on the parameters in `AmrInfo`.  Then for consistence, in `InitAmrMesh`, we should change from `get` to `queryAdd`.